compiler/compile-config/compile-config-2.01-grounding

```diff
@@ -427,16 +375,27 @@ def withRedefinitionsWithJoinsForIdColumns:
 
             # materialize factors using user input_query that pulls in assigned ids to involved variables
             deepdive create table \(.factorsTable | @sh) as \(
-                # allow .input_query to refer to the assigned variable ids by redefining the variable table names with SQL CTE
+                # Not using DISTINCT: User decides if they want to have duplicate factors
```

Comment:
Okay.  I have a feeling it'll waste time&space to materialize what's going to be immediately projected out, but we can do this comparison more rigorously and decide later.